src/renderer/components/state-table.tsx

```diff
@@ -60,6 +74,27 @@ export class StateTable extends React.Component<
     if (this.isStateFile(selectedLogFile)) {
       this.parse(selectedLogFile);
     }
+
+    // Need to find root-state and external-config files when mounted to store data in state
+    const files = this.props.state.processedLogFiles?.state;
+    const searchRootState = 'root-state.json';
+    const searchExternalConfig = 'external-config.json';
+    if (files) {
+      const foundExternalFile = files.find((file) =>
+        file.fileName
+          .toLowerCase()
+          .endsWith(searchExternalConfig.toLowerCase()),
+      );
+      const foundRootFile = files.find((file) =>
```

Comment:
nit: more accurate name
```suggestion
      const foundRootStateFile = files.find((file) =>
```